edx_notifications/const.py

```diff
@@ -24,3 +24,9 @@
 
 NOTIFICATION_BULK_PUBLISH_CHUNK_SIZE = getattr(settings, 'NOTIFICATION_BULK_PUBLISH_CHUNK_SIZE', 100)
 NOTIFICATION_MINIMUM_PERIODICITY_MINS = getattr(settings, 'NOTIFICATION_MINIMUM_PERIODICITY_MINS', 60)  # hourly
+
+PURGE_READ_NOTIFICATIONS_OLDER_THAN_DAYS = getattr(settings, 'PURGE_READ_NOTIFICATIONS_OLDER_THAN_DAYS', None)
```

Comment:
Can we start these configuration settings with the word NOTIFICATION_ just to keep the naming convention consistent. However, we might start to run up against some pylint/pep8 variable length limits, so keep an eye on that. Thanks.